R/csv.R

```diff
@@ -29,8 +29,8 @@
 #' dbReadTable(con, "data")
 #'
 #' dbDisconnect(con)
-duckdb_read_csv <- function(conn, name, files, header = TRUE, na.strings = "", nrow.check = 500,
-                            delim = ",", quote = "\"", col.names = NULL, lower.case.names = FALSE, sep = delim, transaction = TRUE, ...) {
+duckdb_read_csv <- function(conn, name, files, ..., header = TRUE, na.strings = "", nrow.check = 500,
```

Comment:
We'd want to check that the ellipsis is actually empty. This is touching the same file, but the move of the ellipsis and the check could also be a separate PR.

Reply:
I have added a warning. Is it what you wanted?